skcosmo/feature_selection/__init__.py

```diff
@@ -7,6 +7,6 @@
 from .simple_cur import CUR
 from .pcov_fps import PCovFPS
 from .pcov_cur import PCovCUR
+from .voronoi_fps import VoronoiFPS
```

Comment:
Why is this in feature selection instead of sample selection? My understanding (and benchmarks agree) was that Voronoi FPS works best for sample selection, and can be slower than simple fps for features selections.

Reply:
In fact, it's true, yes. And in general it makes sense to move this to sample_selection